### ISSUE TITLE
Add a view for creating tags

### ISSUE THREAD

**Issue by ryanrightmer** [open-austin/BASTA-tfwa#112]
Add a view for creating and editing tags

**Comment by cdpadilla42** [comment]
I'm interested in working on this! If I'm remembering right, this would be for tagging images with descriptions such as "damage", "document", etc. For this view, are we looking to set it up similar to the Github labels interface where you have all the available tags you can edit and add, or is this meant to be the component that goes with images to add and remove tags?

**Comment by ryanrightmer** [comment]
Yes, I think that interface would be good inspiration! We'll also need to create a ticket for the backend work
